test/itex/test_tensorflow_itex_basic.py

```diff
@@ -254,12 +262,15 @@ def test_itex_benchmark_gpu(self):
 
             evaluator = Benchmark('fake_yaml_2.yaml')
             evaluator.b_dataloader = common.DataLoader(dataset)
+            num_of_instance = evaluator.conf.usr_cfg.evaluation.performance.configs.num_of_instance
```

Comment:
sorry, I don't understand why this UT needs to be updated? it doesn't work before?

at INC 2.0 we are focusing on new API. I suppose such old UT shouldn't be updated, why such change is needed?

Reply:
This UT will check the log file name. Sometimes it will fail if the test folder has unexpected '.log' file. This fix is used to avoid such failure.